tests/integration/test_apigateway.py

```diff
@@ -275,7 +275,7 @@ def test_api_gateway_kinesis_integration(self):
         # post test data to Kinesis via API Gateway
         result = requests.post(url, data=json.dumps(test_data))
         result = json.loads(to_str(result.content))
-        assert 0 == result["FailedRecordCount"]
+        assert None == result.get["FailedRecordCount"]
```

Comment:
This is null now. According to the Kinesis API, FailedRecordCount must be > 0 if it is returned, otherwise it is null.